src/plugins/data_source_management/public/components/create_data_source_wizard/components/create_form/create_data_source_form.test.tsx

```diff
@@ -106,7 +106,7 @@ describe('Datasource Management: Create Datasource form', () => {
 
   test('should throw validation error when title is not valid & remove error on update valid title', () => {
```

Comment:
it failed again: https://github.com/opensearch-project/OpenSearch-Dashboards/actions/runs/3064026313/jobs/4946739303 

I think the only allowed currently is https://test.com.

Or you might need to update this file: .lycheeexclude to add dummy urls needed in test.

Reply:
let's validate before merging, you can enable the link test in CI by adding `feature branch`. Then post PR, it will be triggered. After you validate it, you can revert back the change

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.github/workflows/links_checker.yml#L7-L10
@mpabba3003

Reply:
@kristenTian The run you mentioned is old and does not include my changes form this PR

Reply:
I did re-ran the flow with this lasted commit, the link could be outdated, but If you check the PR you can see the latest one still fails.